src/Apps/IdentityVerification/IdentityVerificationApp/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { IdentityVerificationApp_me } from "__generated__/IdentityVerificationApp_me.graphql"
+import { AppContainer } from "Apps/Components/AppContainer"
+import { ErrorPage } from "Components/ErrorPage"
+import { Router } from "found"
+import React from "react"
+import { Title as HeadTitle } from "react-head"
+import { createFragmentContainer, graphql, RelayProp } from "react-relay"
+import { CompleteFailed } from "./CompleteFailed"
+import { CompletePassed } from "./CompletePassed"
+import { StartIdentityVerification } from "./StartIdentityVerification"
+
+interface Props {
+  me: IdentityVerificationApp_me
+  relay: RelayProp
+  router: Router
+}
+
+const IdentityVerificationApp: React.FC<Props> = ({ me, relay, router }) => {
+  const { identityVerification } = me
+
+  if (!identityVerification || identityVerification.userID !== me.internalID) {
```

Comment:
You don't have to change this as part of this PR, but It's odd that you have to do this on the front-end when the `identityVerification` field is a nested field of `me`. The front-end should only check the presence of `identityVerification` and the back-end should take care of hiding `identityVerification`s that belongs to someone else.

Reply:
I agree with you technically - but we had many long conversations and about this approach last month and I don't think it is going to change; a `me.identityVerification` is an idv that the user has access to, not necessarily that _belongs to_ the user, so we do need this check on the client.

Reply:
If I am remembering context correctly, it's worth noting that this is _just_ to protect against the case of an admin accessing this page for an IDV that isn't theirs. At an API layer, we don't allow regular user A to query for the identity verifications for regular user B. In that case, the identity_verification would be `null` (the LHS of this expression).

cc @yuki24